src/components/EventCard.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { ReactElement } from "react";
+import ".././styles.css";
+import PinSVG from ".././public/Pin.svg";
+import TextIconSVG from ".././public/TextIcon.svg";
+import MeatballMenuSVG from ".././public/MeatballMenu.svg";
+
+export interface EventCardProps {
+  startTime: Date;
+  endTime?: Date;
+  name: string;
+  location: string;
+  description: string;
+  live: boolean;
+}
+
+/**
+ * Renders a single event in the feed
+ */
+const EventCard = ({
+  startTime,
+  endTime,
+  name,
+  location,
+  description,
+  live,
+}: EventCardProps): ReactElement => {
+  let startTimeString: string = startTime.toLocaleString("en-US", {
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  });
+  let endTimeString: string | undefined = endTime
+    ? endTime.toLocaleString("en-US", {
+        hour: "numeric",
+        minute: "numeric",
+        hour12: true,
+      })
+    : undefined;
+
+  return (
+    <>
+      <div className="pl-4 pt-4 pr-6 w-4/5">
+        <span className="font-sans">
+          {startTimeString + (endTime ? " to " + endTimeString : "")}
+        </span>
+        <div className="flex flex-row justify-between items-start mb-4">
+          <div className="not-italic font-bold text-2xl leading-8 font-sans break-words w-4/5">
+            {name}
+          </div>
+          <img
+            src={MeatballMenuSVG}
+            alt="Menu svg"
+            aria-label="Open Event Card details"
+          />
+        </div>
+        <div className="flex flex-row items-start mb-4">
+          <img src={PinSVG} alt="Pin svg" className="p-1 pt-0" />
+          <span className="text-body-mobile pl-2 pr-12 pt-0.5 font-montserrat break-words w-full">
+            {location}
+          </span>
+        </div>
+        <div className="flex flex-row items-start mb-6">
+          <img src={TextIconSVG} alt="TextIcon svg" className="p-1 pt-0" />
+          <p className="text-body-mobile pl-2 pr-12 font-montserrat break-words w-full">
+            {description}
+          </p>
+        </div>
+
+        <div className="flex flex-row flex-wrap">
```

Comment:
Looks good! My only suggestion is to make the buttons aligned to the right of the event card similar to the figma wireframe !

Reply:
Soooo I decided to just go ahead and implement the desktop design as well since I won't be here at GM anyways, would you mind taking another look at the desktop changes 😭

If you'd prefer to merge in the mobile implementation first to use for the homepage ticket, lmk and I can do that.

Reply:
Ah is it possible you could push the changes for desktop into another PR? I'd want to look through the desktop changes thoroughly and not rushed, but I also don't want to delay merging this PR into main too much! Lmk if you will need any help moving those changes into another PR!